app/services/cleaners/litigator_claim_cleaner.rb

```diff
@@ -0,0 +1,17 @@
+module Cleaners
+  class LitigatorClaimCleaner < BaseClaimCleaner
+    private
+
+    def destroy_invalid_fees
+      return if case_type.blank?
+
+      if case_type.is_fixed_fee?
+        graduated_fee&.destroy
+        claim.graduated_fee = nil
+      else
+        fixed_fee&.destroy
+        claim.fixed_fee = nil
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/4252/files#r720222724)